include/cuco/detail/bloom_filter/bloom_filter_impl.cuh

```diff
@@ -198,7 +198,7 @@ class bloom_filter_impl {
     auto const grid_size =
       cuco::detail::grid_size(num_keys, cg_size, cuco::detail::default_stride(), block_size);
 
-    detail::add_if_n<cg_size, block_size>
+    bloom_filter_ns::detail::add_if_n<cg_size, block_size>
```

Comment:
`cuco::bloom_filter_ns::detail` or `cuco::detail::bloom_filter_ns`? I think the latter makes it more clear that these `_ns` namespaces are not meant to be publicly available.

Reply:
Makes sense. Updated.